project/src/components/pages/add-review/add-review.jsx

```diff
@@ -1,23 +1,49 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import {useParams} from 'react-router-dom';
 
-import {PosterModifier} from '../../../const';
+import {DATA_LOADING, PosterModifier} from '../../../const';
+import {fetchFilm} from '../../../store/api-actions';
 
+import PageHeader from '../../ui/page-header/page-header';
 import Preview from '../../ui/film-card/preview/preview';
 import Poster from '../../ui/film-card/poster/poster';
 import ReviewForm from '../../ui/review-form/review-form';
+import PageLoading from '../../ui/loading/page-loading/page-loading';
+
+function AddReview({name, posterImage, backgroundImage, backgroundColor, loadFilm}) {
+  const {id} = useParams();
+  const [dataLoading, setDataLoading] = useState(DATA_LOADING);
+
+  const checkDataLoading = (isError = false) => {
+    setDataLoading({isLoading: isError, isError});
+  };
+
+  useEffect(() => {
+    loadFilm(+id)
+      .then(() => checkDataLoading())
```

Comment:
```suggestion
      .then(checkDataLoading)
```

уже не раз писал, зачем создавать новую функцию, которая вызывает другую, ничего ей не передавая. Передай уже сразу ссылку на нужную функцию